jsx/Filter.js

```diff
@@ -88,6 +97,7 @@ class Filter extends Component {
           legend={this.props.title}
         >
           {this.renderFilterFields()}
+          <br/>
```

Comment:
:(

Reply:
Why can't you make this a div?

Reply:
Because i don't think that'll necessarily send the clear filter button to the next line. But i'll put in a div!

Reply:
why would wrapping it in a div not send it to the next line? You could always add top margin to the div if it's not moving it enough.

Reply:
I like @driusan suggestion. You also could use display: block for the div and when two divs are side by side, it will do the similar functionality of putting the divs on separate lines. That's my personal taste besides instead of <br/>.